cmd/goquery_completion/conditional.go

```diff
@@ -152,11 +251,28 @@ func conditional(args []string) []string {
 
 	next := func(tokens []string) suggestions {
 		var suggs []suggestion
-		for _, sugg := range nextAll(antepenultimate(tokens), penultimate(tokens), openParens(tokens)) {
-			if strings.HasPrefix(sugg.token, last(tokens)) {
-				suggs = append(suggs, sugg)
+		prevprev := antepenultimate(tokens)
+		prev := penultimate(tokens)
+		openParens := openParens(tokens)
```

Comment:
Move this line down to 261, so we avoid searching the entire token list in case of an invalid condition string.